packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -19,12 +19,15 @@
   margin-inline: var(--MarginInline);
   padding-block: var(--PaddingBlock);
   padding-inline: var(--PaddingInline);
-  transition-property: color, background-color, border-color, box-shadow;
+  transition-property: color, background-color, border-color, outline-offset,
+    outline-color;
   transition-duration: var(--TransitionDuration);
   transition-timing-function: var(--TransitionTimingFunction);
   border-width: var(--BorderWidth);
   border-style: var(--BorderStyle);
   border-color: var(--PrimaryBorderColor);
+  outline-color: transparent;
```

Comment:
I didn't tokenize these, as they provide the baseline for enabling the transition. Folks *may* want to theme this behavior, but it seems unlikely.